### ISSUE TITLE
"Use expression body for properties" shows zero results when you attempt to run across document, project & solution

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#27947]
**Version Used**: 
Version 15.8.0 Preview 4.0 [27815.3001.d15.8stg]

**Steps to Reproduce**:

``` C#

    public static class ManagedImageMonikers
    {
        private static readonly Guid s_manifestGuid = new Guid("{259567C1-AA6B-46BF-811C-C145DD9F3B48}");
        private const int CodeInformationPrivateId = 2;

        public static ImageMoniker CodeInformationPrivate
        {
            get { return new ImageMoniker { Guid = s_manifestGuid, Id = CodeInformationPrivateId }; } // <-- Attempt to "Use expression body for properties" across document/solution/project
        }
    }

    public class ImageMoniker
    {
        public Guid Guid { get; internal set; }
        public int Id { get; internal set; }
    }
```

**Expected Behavior**:
To be able to run across document, project & solution

**Actual Behavior**:

![image](https://user-images.githubusercontent.com/1103906/41515192-888df9ca-72f0-11e8-93e6-7a4d9e9902a6.png)

**Comment by sharwell** [comment]
Fixes #19089

**Comment by sharwell** [comment]
:memo: I'm trying to determine what the status of d15.8stg is. This is already fixed in d15.8rel but you're working from a different branch.

**Comment by davkean** [comment]
@sharwell That bug you dup'd against has to do with editorconfig. There was no editorconfig involved.

**Comment by sharwell** [comment]
@davkean Ah, great! Then this is probably a follow-up issue to the work in #27220, specifically addressing cases where some of the IDE diagnostics applied self-filtering to make Hidden severity behave like None (i.e. prior to the design change implemented by #27220, some of the diagnostics implemented the design at a per-diagnostic granularity, resulting in confusing experiences like you've observed).

**Comment by sharwell** [comment]
Fixed by #28043
